site/docs/v1/tech/shared/_securing_http_requests.adoc

```diff
@@ -30,11 +30,11 @@ router.route('/fusionauth-{request_entity_lc}').post((req, res) => {
 });
 ----
 
-=== Client Certificates
+=== Certificates
 
-You may create a client certificate to secure your {request_entity}. {ssl_certificate_sentence}. It is used to establish a TLS connection to the {request_entity}. 
+You may create a certificate to use with your {request_entity}. {ssl_certificate_sentence}. It is used to establish a TLS connection to the {request_entity} endpoint. 
```

Comment:
I wonder if a better way to say this is that you can provide an x.509 certificate in PEM format for use on this request. 

While, it is true - you could create your own cert (self signed) and then use it to secure the connection to your web hook, I would assume a more common scenario is that you do have an HTTPS connection already and you may need to provide an existing certificate to allow a connection to work correctly. 

The first choice is to not use this option - and only use it if FusionAuth cannot otherwise connect w/out it.

Reply:
Reworded.